src/Tools/ExternalAccess/FSharp/InternalAPI.Unshipped.txt

```diff
@@ -593,6 +593,7 @@ static Microsoft.CodeAnalysis.ExternalAccess.FSharp.TaskList.FSharpTaskListItem.
 ~static Microsoft.CodeAnalysis.ExternalAccess.FSharp.ExternalAccessFSharpResources.TheValueIsUnused.get -> string
 ~static Microsoft.CodeAnalysis.ExternalAccess.FSharp.ExternalAccessFSharpResources.UnusedOpens.get -> string
 ~static Microsoft.CodeAnalysis.ExternalAccess.FSharp.FindUsages.FSharpDefinitionItem.Create(System.Collections.Immutable.ImmutableArray<string> tags, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.TaggedText> displayParts, Microsoft.CodeAnalysis.ExternalAccess.FSharp.FSharpDocumentSpan sourceSpan) -> Microsoft.CodeAnalysis.ExternalAccess.FSharp.FindUsages.FSharpDefinitionItem
+~static Microsoft.CodeAnalysis.ExternalAccess.FSharp.FindUsages.FSharpDefinitionItem.CreateNonNavigableItem(System.Collections.Immutable.ImmutableArray<string> tags, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.TaggedText> displayParts, System.Collections.Immutable.ImmutableArray<(string name, System.Version version, string filePath)> metadataLocations) -> Microsoft.CodeAnalysis.ExternalAccess.FSharp.FindUsages.FSharpDefinitionItem
```

Comment:
does f# actually need this?  i'd just leave out if not.

Reply:
It is referenced.

Reply:
i'm a tad bit confused.  how can F# reference this, if it's a new method?